examples/server/server.py

```diff
@@ -23,7 +23,10 @@ def __init__(self, transform):
         self.transform = transform
 
     async def recv(self):
-        frame = await self.received.get()
+        try:
+            frame = await asyncio.wait_for(self.received.get(), 1)
+        except asyncio.TimeoutError:
```

Comment:
Let's not commit these changes, if you want to send bogus video keep the hacks on your laptop :)